src/monodroid/monodroid.csproj

```diff
@@ -32,6 +32,11 @@
       <Name>dependencies</Name>
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
+    <ProjectReference Include="..\..\external\Java.Interop\tools\jnimarshalmethod-gen\Xamarin.Android.Tools.JniMarshalMethodGenerator.csproj">
```

Comment:
It is here to make sure we have $(XAInstallPrefix)xbuild\Xamarin\Android\Java.Runtime.Environment.dll.config before modifying it.

Reply:
Given you have a `sed` -- and/or `<ReplaceFileContents/>` -- which can have *any* input path, why does `$(XAInstallPrefix)xbuild\Xamarin\Android\Java.Runtime.Environment.dll.config` need to exist before it's modified?

You could do, `sed`-wise:

```
sed regex... source-file > dest-file
```